leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -452,35 +302,20 @@ var ElementInspector = Backbone.View.extend({
     updateHTML: function(event) {
         // Check if selected init sat value and functionType pair is illegal
         // Only runs if evolvingFunction is defined and therefore there is a function type
-        // if (this.intention.get('evolvingFunction') != null) {
```

Comment:
Do we not need this anymore?

Reply:
Yeah, I think that was a quick fix for something so it's no longer needed